style.css

```diff
@@ -0,0 +1,108 @@
+body {
+    background-color: #fff;
+    width: 100%;
+    font-family: 'Proxima Nova', Georgia, sans-serif;
+    margin: 0 auto;
+}
+
+
+header {
+    max-width: 100%;
+    height: 567px;
```

Comment:
Better to limit height with some max value instead.
On mobile seems designer wanted hero image to fill full screen - so try to use `100vh`.